test/normalize.py

```diff
@@ -66,7 +66,7 @@ def handle_starttag(self, tag, attrs):
                     self.output += ("=" + '"' +
                             urllib.quote(urllib.unquote(v), safe='/') + '"')
                 elif v != None:
-                    self.output += ("=" + '"' + cgi.escape(v,quote=True) + '"')
+                    self.output += ("=" + '"' + html.escape(v) + '"')
```

Comment:
(`quote=True` is the default for `html.escape()`)

Reply:
I see, thanks